### ISSUE TITLE
Does not scan SHTML files in the repo

### ISSUE THREAD

**Issue by meneguzzi** [celinekurpershoek/link-checker#4]
I just installed the action for my personal website (the repo is private, but I can add you if you want to use it as a test case), but it did not detect broken links. I assume this is because the file patterns it follows do not include the ```shtml``` extension.  

I may be missing some configuration parameters here, but it would be nice if there was a parameter that lets the user specify included file patterns explicitly.

**Comment by meneguzzi** [comment]
My stupidity configuring it, sorry about that.
